src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable consistent-return */
+import getBooks from '../../api/api';
+
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = getBooks();
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
```

Comment:
- [ ] when adding a book it doesn't actually update the Redux store with the new book so after refreshing the the page the placeholder books are shown again and the added books are not visible anymore.